database/migrations/2020_05_07_180151_add_plant_type_id_to_species_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddPlantTypeIdToSpeciesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('species', function (Blueprint $table) {
+            $table->unsignedBigInteger('plant_type_id')->nullable();
+
+            $table->foreign('plant_type_id')
+                ->references('id')
+                ->on('plant_types')
+                ->onDelete('cascade');
```

Comment:
Since it's nullable, let's set null on delete instead of cascading.
```suggestion
                ->onDelete('set null');
```